### PR TITLE
schema: split JSON schemas

### PR DESCRIPTION
**Pull request by rerowep**
* Splits document schema into small files.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: rero_ils/jsonschemas/utils.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+class JsonLoader(JsonRefLoader):
+    """Json schema loader.
+
+    We have to add local $ref loading to the base class.
+    https://invenio-jsonschemas.readthedocs.io/en/latest/configuration.html
+    """
+
+    @lru_cache(maxsize=1000)
+    def get_remote_json(self, uri, **kwargs):
+        """Get remote json.
```
Comment (jma) [inline rero_ils/jsonschemas/utils.py:38]: Parameters are not described in the documentation.

File: tests/utils.py [hunk 2]
```diff
@@ -187,3 +190,52 @@ def check_timezone_date(timezone, date, expected=[]):
...
+        return MockResponse({}, 404)
+    schema = json.loads(schema_in_bytes.decode('utf8'))
+    if not schema:
+        return MockResponse({}, 404)
+
+    return MockResponse(schema, 200)
+
+
+def get_schema(monkeypatch, schema_in_bytes):
+    """Get Jsonschema and replace $refs."""
```
Comment (jma) [inline tests/utils.py:233]: Parameters are not described.

File: tests/utils.py [hunk 3]
```diff
@@ -187,3 +190,52 @@ def check_timezone_date(timezone, date, expected=[]):
         assert hour in expected, error_msg
     assert tocheck_date.minute == date.minute, error_msg
     assert tocheck_date.hour == hour, error_msg
+
+
+def mocked_requests_get(*args, **kwargs):
+    """This method will be used by the mock to replace requests.get."""
```
Comment (jma) [inline tests/utils.py:196]: Can you add more details in the documentation. What this class do?

**Comment by zannkukai** [comment]
Don't forget to extract_messages/update_catalog to update line numbers into translations files

**Pull request merged**